udata/search/fields.py

```diff
@@ -56,7 +58,20 @@ def get_value_filter(self, value):
 
 
 class TermsFacet(Facet, DSLTermsFacet):
-    pass
+
+    def add_filter(self, filter_values):
+        if not filter_values:
```

Comment:
Alright, I'm interested in concrete tests that are not addressed by the current PR. Be aware that the return of the `add_filter` function should be something `clone`able like a `Q()` object.